### DISCUSSION TITLE
빌드잇 맵 사용하기

### DISCUSSION THREAD

**Discussion by smilejsu82** [naverz/zepeto-studio-kor#529] [5. Q&A]
내가 만든 맵이 안보여요 

맵을 공개로 바꿔야 하나요?

![image](https://user-images.githubusercontent.com/50188795/159978608-6e29cebe-e92d-4ca0-9923-20c6e9c22f61.png)

![image](https://user-images.githubusercontent.com/50188795/159978573-e242bb40-b226-44dc-bb1e-b1f65d49328b.png)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 테스트를 위해 새로 월드를 만들고 빌드잇 로드를 실행시켰는데 저는 잘 보여집니다. 
출시되지 않은 맵도 보여져야 정상입니다.
유니티 상에 로그인은 잘 되어 있는지 한번 체크해 보세요.
